README.md

```diff
@@ -106,9 +127,9 @@ Integration tests use the `test_context` [pytest fixture](https://docs.pytest.or
 The entire suite of tests can be run with the following commands from the top level project directory:
 
     export OPENREVIEW_HOME=<path_to_openreview>
-    python -m pytest tests
+    python -m pytest -s tests
```

Comment:
what is `-s` for?

Reply:
It sets env_variable `PYTHONNOUSERSITE` to True. I was having issues running tests locally. but tests were passing on CircleCI. CircleCI config has this flag as well, so I tried this and it worked.

Reply:
the pytest docs says:
```
  --capture=method      per-test capturing method: one of fd|sys|no.
  -s                    shortcut for --capture=no.

```

When I use this parameter is printing a lot of output from the openreview-py that I would prefer not to see. 

Could you remove it from the README file?, this parameter is not needed to run the tests.

If you still have issues running the tests locally, we can talk about it.

Reply:
Done.